.gitignore

```diff
@@ -9,3 +9,5 @@ vendor
 
 # Intellij
 .idea/
+
+.vscode/
```

Comment:
For future reference, I realized I could stop adding `.idea/` to all our public repos, and instead add a `.gitignore` file in the `.idea/` directory, containing:
```
*.*
*/**/*.*
```

Reply:
Or you use your local `.gitignore` file

Reply:
kk, will take it out